CHANGELOG

```diff
@@ -40,6 +40,10 @@ Master, using release name V 2.4.0 (1/7/25)
 * Add options for user-provided FFTW locker (PR548, Blackwell). These options
   can be be used to prevent crashes when a user is creating/destroying FFTW
   plans and FINUFFT plans in threads simultaneously.
+* Make attributes private in Python Plan classes and allow read-only access to
+them using properties.
```

Comment:
If you can add the PR #s that would be great. Still wrap lines at 80 chars.

Reply:
Also I've tended to add the new stuff at the top not the bottom of the current list.